src/SDKs/ApiManagement/Management.ApiManagement/Microsoft.Azure.Management.ApiManagement.csproj

```diff
@@ -8,7 +8,7 @@
     <Description>Provides ApiManagement management capabilities for Microsoft Azure.</Description>
     <AssemblyTitle>Microsoft Azure API Management Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.ApiManagement</AssemblyName>
-    <Version>4.0.2-preview</Version>
+    <Version>4.0.3-preview</Version>
```

Comment:
Please update the `AssmeblyFileVersion` to `4.0.3` [here](https://github.com/solankisamir/azure-sdk-for-net/blob/242f567665422b0005716f1d138a0ef4faa2cfbb/src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs)

Reply:
Fixed